platform-includes/getting-started-sourcemaps/javascript.react-router.mdx

```diff
@@ -0,0 +1,9 @@
+## Add Readable Stack Traces to Errors
+
+By default, React Router will minify your JavaScript and CSS files in production. This makes it difficult to debug errors in production. To make debugging easier, you can generate source maps and upload them to Sentry.
+
+We recommend to use [Sentry's Vite plugin](/platforms/javascript/sourcemaps/uploading/vite/) to upload sourcemaps.
+
+Please refer to the <PlatformLink to="/sourcemaps">Sourcemaps Documentation</PlatformLink>, for more information.
```

Comment:
```suggestion
Please refer to the <PlatformLink to="/sourcemaps">Source Maps Documentation</PlatformLink>, for more information.
```

In most of our docs we use 2 words for 'source maps', let's stick to that convention 🤙